README.md

```diff
@@ -30,6 +33,20 @@ npm start
 
 Please read the [Contributing guidelines](.github/CONTRIBUTING.md), [Code of Conduct](https://github.com/anitab-org/open-source-programs-web/blob/develop/CODE_OF_CONDUCT.md) and [Reporting Guidelines](https://github.com/anitab-org/open-source-programs-web/blob/develop/REPORTING_GUIDELINES.md)
 
+Remember to run eslint and prettier before pushing the changes.
+
+### To run eslint
+
+```sh
+npm run lint:fix
+```
+
+### To format files
+
+```sh
+npm run format
+```
```

Comment:
Hey @jalajcodes you could remove the sh from ```sh that's not required

Reply:
sure!